docs/core/diagnostics/dotnet-sos.md

```diff
@@ -51,7 +51,7 @@ The `dotnet-sos` global tool installs the [SOS debugger extension](../../framewo
 
 ## dotnet-sos install
 
-Installs the [SOS extension](../../framework/tools/sos-dll-sos-debugging-extension.md) locally for debugging .NET Core processes. On macOS and Linux, the .lldbinit file will be updated so that the extension automatically loads at lldb startup. If installing SOS on Windows with older debugging tools (< version 10.0.18317.1001), you will need to manually load the extension in WinDbg or cdb by running `.load %USERPROFILE%\.dotnet\sos\sos.dll` in the debugger.
+Installs the [SOS extension](sos-debugging-extension.md) locally for debugging .NET Core processes. On macOS and Linux, the .lldbinit file will be updated so that the extension automatically loads at lldb startup. If installing SOS on Windows with older debugging tools (< version 10.0.18317.1001), you will need to manually load the extension in WinDbg or cdb by running `.load %USERPROFILE%\.dotnet\sos\sos.dll` in the debugger.
```

Comment:
This might want to be split into multiple paragraphs/sections to make it easy to quickly parse.  Also these implementation details might not be relevant to the documentation.

I am not sure this is placed correctly.  I suspect it belongs after the synopsis.  But I haven't looked at the style guide recently.